test/unit/verifier/lint/test_ansible_lint.py

```diff
@@ -0,0 +1,209 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to
+#  deal in the Software without restriction, including without limitation the
+#  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+#  sell copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+
+import pytest
+import sh
+
+from molecule import config
+from molecule.verifier.lint import ansible_lint
+
+
+@pytest.fixture
+def _verifier_lint_section_data():
+    return {
+        'verifier': {
+            'name': 'ansible',
+            'lint': {
+                'name': 'ansible-lint',
+                'options': {
+                    'foo': 'bar',
+                    'v': True,
+                    'exclude': [
+                        'foo',
+                        'bar',
+                    ],
+                    'x': [
+                        'foo',
+                        'bar',
+                    ],
+                },
+                'env': {
+                    'FOO': 'bar',
+                },
+            }
+        }
+    }
+
+
+# NOTE(retr0h): The use of the `patched_config_validate` fixture, disables
+# config.Config._validate from executing.  Thus preventing odd side-effects
+# throughout patched.assert_called unit tests.
+@pytest.fixture
+def _instance(patched_config_validate, config_instance):
+    return ansible_lint.AnsibleLint(config_instance)
+
+
+def test_config_private_member(_instance):
+    assert isinstance(_instance._config, config.Config)
+
+
+def test_default_options_property(_instance):
+    x = {
```

Comment:
This is optional BUT if you want to use more indicative naming, that'd be great :+1: